README.md

```diff
@@ -1,306 +1,184 @@
-   itself one level deeper into a `testnet` subfolder (`~/.ethereum/testnet` on Linux). Note, on OSX
-   and Linux this also means that attaching to a running testnet node requires the use of a custom
-   endpoint since `geth attach` will try to attach to a production node endpoint by default. E.g.
-   `geth attach <datadir>/testnet/geth.ipc`. Windows users are not affected by this.
- * Instead of connecting the main Ethereum network, the client will connect to the test network,
-   which uses different P2P bootnodes, different network IDs and genesis states.
+Sharding Instructions
+==============
 
-*Note: Although there are some internal protective measures to prevent transactions from crossing
-over between the main network and test network, you should make sure to always use separate accounts
-for play-money and real-money. Unless you manually move accounts, Geth will by default correctly
-separate the two networks and will not make any accounts available between them.*
+To get started with running the project, follow the instructions to initialize your own private Ethereum blockchain and geth node, as they will be required to run before you can become a validator or a proposer.
 
-### Full node on the Rinkeby test network
+Running a Local Geth Node
+------------------------------
 
-The above test network is a cross client one based on the ethash proof-of-work consensus algorithm. As such, it has certain extra overhead and is more susceptible to reorganization attacks due to the network's low difficulty / security. Go Ethereum also supports connecting to a proof-of-authority based test network called [*Rinkeby*](https://www.rinkeby.io) (operated by members of the community). This network is lighter, more secure, but is only supported by go-ethereum.
+To start a local Geth node, you can create your own `genesis.json` file similar to:
 
+```json
+{
+    "config": {
+        "chainId": 12345,
+        "homesteadBlock": 0,
+        "eip155Block": 0,
+        "eip158Block": 0
+    },
+    "difficulty": "200",
+    "gasLimit": "210000000000"
+}
 ```
-$ geth --rinkeby console
-```
-
-### Configuration
 
-As an alternative to passing the numerous flags to the `geth` binary, you can also pass a configuration file via:
+Then, you can build `geth` and init a new instance of a local, Ethereum blockchain as follows:
 
 ```
-$ geth --config /path/to/your_config.toml
+$ make geth
+$ ./build/bin/geth init ./sharding/genesis.json -datadir /path/to/your/datadir
+$ ./build/bin/geth --nodiscover console --datadir /path/to/your/datadir --networkid 12345
 ```
-
-To get an idea how the file should look like you can use the `dumpconfig` subcommand to export your existing configuration:
+Then, the geth console can start up and you can start a miner as follows:
 
 ```
-$ geth --your-favourite-flags dumpconfig
+> personal.newAccount()
+> miner.setEtherbase(eth.accounts[0])
+> miner.start()
```

Comment:
I would recommend setting the core number to 1. i.e. `miner.start(1)`. That why you're only churning a fraction of your total CPU